gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/service/ExportCornerComputationService.java

```diff
@@ -44,33 +45,30 @@
 @Service
 public class ExportCornerComputationService {
 
+    private final ComputationService computationService;
     private final DichotomyRunner dichotomyRunner;
     private final FileImporter fileImporter;
     private final FileExporter fileExporter;
     private final Logger businessLogger;
-    private final CseValidNetworkShifter cseValidNetworkShifter;
+    private final CseValidNetworkShifterProvider cseValidNetworkShifterProvider;
     private final CseValidRaoValidator cseValidRaoValidator;
 
-    public ExportCornerComputationService(DichotomyRunner dichotomyRunner,
+    public ExportCornerComputationService(ComputationService computationService,
+                                          DichotomyRunner dichotomyRunner,
                                           FileImporter fileImporter,
                                           FileExporter fileExporter,
                                           Logger businessLogger,
-                                          CseValidNetworkShifter cseValidNetworkShifter,
+                                          CseValidNetworkShifterProvider cseValidNetworkShifterProvider,
                                           CseValidRaoValidator cseValidRaoValidator) {
+        this.computationService = computationService;
         this.dichotomyRunner = dichotomyRunner;
         this.fileImporter = fileImporter;
         this.fileExporter = fileExporter;
         this.businessLogger = businessLogger;
-        this.cseValidNetworkShifter = cseValidNetworkShifter;
+        this.cseValidNetworkShifterProvider = cseValidNetworkShifterProvider;
         this.cseValidRaoValidator = cseValidRaoValidator;
     }
 
-    private String generateScaledNetworkDirPath(Network network, OffsetDateTime processTargetDateTime, ProcessType processType) {
-        String basePath = fileExporter.makeDestinationMinioPath(processTargetDateTime, processType, FileExporter.FileKind.ARTIFACTS);
-        String variantName = network.getVariantManager().getWorkingVariantId();
-        return basePath + variantName;
-    }
-
     public void computeTimestamp(TTimestampWrapper timestampWrapper, CseValidRequest cseValidRequest, TcDocumentTypeWriter tcDocumentTypeWriter) {
         TTimestamp timestamp = timestampWrapper.getTimestamp();
```

Comment:
This variable is defined at the very first line of the method, but `timestampWrapper.getTimestamp()` is still used 4 times as well. Better use the variable everywhere.